### ISSUE TITLE
Unable to navigate export menu with NVDA+Firefox

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/highcharts#10543]
There seems to be an issue where NVDA+Firefox fails to navigate the export menu.

The screen reader only announces two items (the separator and everything after is ignored). Seems to be a keyboard trap. Escape does not work, nor does arrow keys.

**Comment by oysteinmoseng** [comment]
Should be working in latest dev code.
